### PR TITLE
Create Search button

### PR DESCRIPTION
**Pull request by J-Bannach**
- Build "Search" button component
- Add component to storybook

### CODE REVIEW

**Review by hugh-burgess — APPROVED** [review]
LGTM! Your teacher has given you a different approach to what I'm used to, but this looks great so far!

File: src/app/components/SearchButton/SearchButton.tsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-type ButtonProps = {
+type SearchButtonProps = {
   children: ReactNode;
   onClick?: () => void;
 };
 
-function Button({ children, onClick }: ButtonProps): JSX.Element {
+function SearchButton({ children, onClick }: SearchButtonProps): JSX.Element {
   return (
     <button className={styles.button} onClick={onClick}>
```
Comment (l-goede) [inline src/app/components/SearchButton/SearchButton.tsx:11]: Would change class name to searchButton

**Review by l-goede — APPROVED** [review]
lgtm :)

File: src/app/components/SearchButton/SearchButton.tsx (hunk 1 above)
Comment (J-Bannach) [reply src/app/components/SearchButton/SearchButton.tsx:11]: Good idea, thanks.

**Pull request merged**